libs/model-assessment/src/lib/ModelAssessmentDashboard/AvailableTabs.ts

```diff
@@ -10,42 +9,26 @@ import { GlobalTabKeys } from "./ModelAssessmentEnums";
 
 export function getAvailableTabs(
   props: IModelAssessmentDashboardProps,
-  jointDataset: JointDataset,
   excludeErrorAnalysis: boolean
 ): IDropdownOption[] {
   const availableTabs: IDropdownOption[] = [];
-  if (
-    !excludeErrorAnalysis &&
-    props.requestDebugML &&
```

Comment:
shouldn't we have at least one of:
-  these methods
- preprocessed data?
Now it looks like we're not checking at all.

Reply:
Technically yes.
But that would not happen, errorAnalysisData only added with either requestDebugML or preprocesseddata
I added extra check for safty.